src/BookSettings.ts

```diff
@@ -42,6 +42,20 @@ const checkSvg = `<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 45 32" pr
 </svg>
 `;
 
+export interface BookSettingsConfig {
+    /** Store to save the user's selections in. */
+    store: Store,
+
+    /** Array of BookView options. */
```

Comment:
It is an array of `BookView`s, and the user can select which one to use in settings. They are "options" for the end user. You're right, this comment wasn't a good explanation.